contracts/wrapped-assets/shiba/WrappedShibaSwapLp.sol

```diff
@@ -228,6 +241,45 @@ contract WrappedShibaSwapLp is IWrappedAsset, Auth2, ERC20, ReentrancyGuard {
         return _sslpToken;
     }
 
+    /**
+     * @notice Withdraw tokens from topdog to user proxy without caring about rewards. EMERGENCY ONLY.
+     * @notice To withdraw tokens from user proxy to user use `withdrawToken`
+     */
+    function emergencyWithdraw() public nonReentrant {
```

Comment:
note: this method does not send tokens to user intentionally - to minimize work done by emergency method